compiler/res/prog8lib/atari/textio.p8

```diff
@@ -21,7 +21,7 @@ sub home() {
 }
 
 sub nl() {
-    txt.chrout('\n')
+    txt.chrout(155)
```

Comment:
This should really be fixed by properly defining the Atari's character encoding.
At first glance it looked like ISO so I used this, but it seems there are differences. Someone has to define the proper atari  character encoding/decoding tables.
We can do this later though and for now, roll with this change as a temporary workaround.

Reply:
Atari 8-bit computers uses so called ATASCII. It works with the OS calls en E: handler.
There are special characters like 155 for END-OF-LINE and other control characters.

https://en.wikipedia.org/wiki/ATASCII

Reply:
what's an E: handler?

Reply:
E: is the Editor device. There are also device handlers for D: (disk drive), C: (cassette), P: (printer) and S: (screen). The E: device is used for all text mode I/O.

https://www.atarimagazines.com/v3n8/AllAbout_IO.html